pkgs/development/python-modules/nengo/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitHub, buildPythonPackage,
+  numpy,
+  scipySupport ? false, scipy,
+  scikitSupport ? false, scikit-learn,
```

generally it's preferred to have leading `,`'s, to avoid larger diffs
```suggestion
{ lib, fetchFromGitHub, buildPythonPackage
, numpy
, scipySupport ? false, scipy
, scikitSupport ? false, scikit-learn
```

---

cc @FRidh on handling optional dependencies